bpf/kmesh/workload/include/service.h

```diff
@@ -17,9 +17,13 @@ static inline int lb_random_handle(struct kmesh_context *kmesh_ctx, __u32 servic
     int ret = 0;
     endpoint_key endpoint_k = {0};
     endpoint_value *endpoint_v = NULL;
+    int rand_k = 0;
 
     endpoint_k.service_id = service_id;
-    endpoint_k.backend_index = bpf_get_prandom_u32() % service_v->endpoint_count + 1;
+    endpoint_k.prio = MIN_PRIO; // for random handle，all endpoints are saved in MIN_PRIO
+
+    rand_k = bpf_get_prandom_u32() % service_v->prio_endpoint_count[MIN_PRIO] + 1;
```

Comment:
Yes, the current EndpointSwap implementation relies on the index starting from 1. If we change it to start from 0, EndpointSwap would encounter issues. I believe this can be addressed as a separate optimization point, or we could simply keep the index starting from 1.